frontend/dogplus-frontend/src/pages/loginPage.tsx

```diff
@@ -13,12 +14,33 @@ export const LoginPage = () => {
     setPassword(event.target.value);
   };
 
-  const handleSubmit = (event: FormEvent) => {
+  const handleSubmit = async (event: FormEvent) => {
     event.preventDefault();
     console.log(
       `Logging in with username: ${username} and password: ${password}`
     );
-    // Handle login logic here
+    try {
+      const response = await fetch("http://localhost:8000/api/auth/login/", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({ username, password }),
+      });
+
+      if (!response.ok) {
+        throw new Error("Login failed");
```

Comment:
Bør gi noe feedback til brukeren her. For eks rød tekst hvor det står "Login failed"

Reply:
Litt som i register pagen hvor vi har rød tekst om passordet er for svakt